teams/compiler-contributors.toml

```diff
@@ -0,0 +1,29 @@
+name = "compiler-contributors"
+subteam-of = "compiler"
+
+[people]
+leads = []
+members = [
+  "flodiebold",
+  "lqd",
+  "matklad",
+  "matthewjasper",
+  "nikic",
+  "nnethercote",
+  "scalexm",
+  "spastorino",
+  "tmandry",
+  "wesleywiser",
+  "Xanewok",
+  "zackmdavis",
+]
+
+[permissions]
+perf = true
```

Comment:
curious: does this imply bors try?

Reply:
Well, the `perf = true` doesn't grant y'all bors perms, but you'll also have `bors.rust.review = true` which means `@bors try` and `@bors r+`.